regexes/device/televisions.yml

```diff
@@ -254,7 +254,7 @@ Inverto:
 
 # JVC
 JVC:
-  regex: 'AFTSO001|JVC[;,]'
+  regex: 'AFTSO001|JVC[;,]|JVC Shell'
```

Comment:
Can a JVC Shell useragent also include HbbTV? Or is this unneeded?

Reply:
no it can't, it's a mistake